src/Modal/ModalContent.jsx

```diff
@@ -10,9 +10,19 @@ import style from './Modal.style'
 
 import { Box } from '@ivoryio/kogaio'
 
+const inIframe = () => {
+  try {
+    return window.self !== window.top
+  } catch (e) {
+    return true
+  }
+}
+const isModalInIframe = inIframe()
+
 const StyledModalContent = styled(Box)(
   compose(layout, color, flexbox, border),
-  css(style.content)
+  css(style.content),
+  isModalInIframe && css({ position: 'absolute', top: [3, 9] })
```

Comment:
do we differentiate mobile vs desktop ?

Reply:
I am reopening this question:

can we set a different top value (like 100-300px or whatever looks good) for the desktop so that it's "closer to the middle of the screen"?